pkg/apis/v1alpha1/vpcnetworkconfiguration_types.go

```diff
@@ -46,6 +46,9 @@ type VPCNetworkConfigurationSpec struct {
 	// +kubebuilder:validation:MaxLength=8
 	// +optional
 	ShortID string `json:"shortID,omitempty"`
+	// NSXTVPC is the NSX-T VPC the Namespace associated with.
```

Comment:
NSX-T -> NSX

Reply:
I saw that `NSXTProject` also uses "NSX-T Project" in the comment. If here I modify as "NSX", do you mean NSXProject also needs update?